### PR TITLE
chore(license): update to LICENSE v3

### PR DESCRIPTION
**Pull request by srl295**
- https://github.com/unicode-org/.github/issues/6

### CODE REVIEW

**Comment by annebright** [comment]
The Copyright & Licenses language is fine, but the LICENSE link appears to be broken. Also, there is still language earlier in the ReadMe referencing the SIL license - this is the correct license, but this language should either be removed or added to the Copyright & Licenses language at the bottom of the ReadMe. You can just remove it so long as the broken LICENSE link is fixed and resolves to the SIL license.

**Comment by srl295** [comment]
> The Copyright & Licenses language is fine, but the LICENSE link appears to be broken. 

I had neglected a commit that renamed LICENSE.md to LICENSE, it's now included.

> Also, there is still language earlier in the ReadMe referencing the SIL license - this is the correct license, but this language should either be removed or added to the Copyright & Licenses language at the bottom of the ReadMe. You can just remove it so long as the broken LICENSE link is fixed and resolves to the SIL license.

I moved it into the Copyright & Licenses section. It seems like it might be worth pointing out since it's a different license?

**Comment by kenlunde** [comment]
This is all fine and dandy, but I had major update to the README.md file staged for Version 15.1 on my local machine, which was being held until 2023-09-12. I wish that I was first consulted before this PR was initiated, because now I will need to manually resolve the differences. Although less critical, because I have nothing staged locally, I wish that I were also first consulted on the other two repos: CJK Symbols and UK-Source Ideographs.

**Comment by srl295** [comment]
> This is all fine and dandy, but I had major update to the README.md file staged for Version 15.1 on my local machine, which was being held until 2023-09-12. I wish that I was first consulted before this PR was initiated, because now I will need to manually resolve the differences. Although less critical, because I have nothing staged locally, I wish that I were also first consulted on the other two repos: CJK Symbols and UK-Source Ideographs.

Hi Ken,

This is being rolled out across Unicode, it's been discussed some in EMG.

I don't think the other repos PRs have been merged either, let me know if you'd like something different.

If you pull down my PR, does it conflict with your changes? Is there something I can do to conflict less?

File: README.md [hunk 1]
```diff
@@ -7,10 +7,6 @@
 * Unassigned (reserved for future assignment)
 * A noncharacter
 
-## License
-
-The use of the *Last Resort* font is covered by the terms of the [SIL Open Font License, Version 1.1](https://scripts.sil.org/OFL) license. 
-
```
Comment (srl295) [inline README.md:13]: @kenlunde @annebright if we leave these lines in, then my change is only adding lines and so shouldn't conflict.

**Comment by srl295** [comment]
@kenlunde any concern on this one?

**Comment by kenlunde** [comment]
Fire away. I will resolve the staged update manually.

**Pull request merged**